src/cloud-element-templates/LinterPlugin.js

```diff
@@ -33,6 +35,10 @@ export const elementTemplateLintRule = ({ templates = [] }) => {
 
   function check(node, reporter) {
 
+    if (is(node, 'bpmn:Message')) {
```

Comment:
To harden this, we could even disable the rule for none-`FlowElement`s

Reply:
Let's do it! https://github.com/bpmn-io/bpmn-js-element-templates/pull/7